Kernel/Net/TCP.h

```diff
@@ -81,15 +81,15 @@ class [[gnu::packed]] TCPPacket {
     void* payload() { return ((u8*)this) + header_size(); }
 
 private:
-    NetworkOrdered<u16> m_source_port;
-    NetworkOrdered<u16> m_destination_port;
-    NetworkOrdered<u32> m_sequence_number;
-    NetworkOrdered<u32> m_ack_number;
-
-    NetworkOrdered<u16> m_flags_and_data_offset;
-    NetworkOrdered<u16> m_window_size;
-    NetworkOrdered<u16> m_checksum;
-    NetworkOrdered<u16> m_urgent;
+    NetworkOrdered<u16> m_source_port { 0 };
```

Comment:
`NetworkOrdered<T>` is default initialised to zero, these don't really need to be default initialised here

Reply:
fixed, thanks!